### PR TITLE
Convert int input to static_argnums to a tuple

### LINKED ISSUE #969: convert integer for `static_argnums` to tuple automatically
I've made the same mistake a few times on `static_argnums` on `jit` function, by not using a tuple 

e.g., `jit(f, static_argnums=1)`
would it be possible / reasonable to add a simple `if` statement to convert it to tuple?

where the correct way is  `jit(f, static_argnums=(1,) )`

also note `(1)` is not a tuple,  but `(1,)` is

### PR DESCRIPTION
**Pull request by zhongwen**
User could make mistake of passing an int to static_argnums, this helps to avoid unnecessary error.

Fix #969

### CODE REVIEW

File: jax/api.py [hunk 1]
```diff
@@ -110,6 +110,9 @@ def _jit(fun, static_argnums, device_values=True):
   def f_jitted(*args, **kwargs):
     if _jit_is_disabled or config.read('jax_disable_jit'):
       return fun(*args, **kwargs)
+    if isinstance(static_argnums, int):
+      # static_argnums is a tuple of ints
+      static_argnums = tuple([static_argnums])
```
Comment (mattjj) [inline jax/api.py]: How about `static_argnums = (static_argnums,)`?

**Review by mattjj — APPROVED** [review]
Great idea, and LGTM! I made a small suggestion (of using a tuple literal), just lmk if you want to make that change or think we should merge as-is.

**Comment by zhongwen** [comment]
change made according to your suggestion!

**Comment by mattjj** [comment]
Thanks!

**Pull request merged**